### DISCUSSION TITLE
PlanetScale not reflecting pushed schema

### DISCUSSION THREAD

**Discussion by katungi** [prisma/prisma#18385] [Q&A]
I have a planetscale database with 2 branches, `staging` and `main`. Main is used in production.
I made a change to staging with the following changes:

```prisma
model Program {
  id          Int      @id @default(autoincrement())
  createdAt   DateTime @default(now())
  updatedAt   DateTime @updatedAt
  name        String
  description String   @db.Text
  units       Unit[]
  steps       Step[]. // Added this line
}

# Added this Table
model Step {
  id        Int      @id @default(autoincrement())
  createdAt DateTime @default(now())
  updatedAt DateTime @updatedAt
  step      Int
  content   String   @db.Text
  programId Int
  program   Program  @relation(fields: [programId], references: [id])

  @@index([programId])
}
```
On pushing this, the Steps table is created, but the `steps[]` relation doesn't seem to have been pushed successfully. It's missing in production despite merging the deploy request from staging to main.

What could I be missing?

**Comment by ludralph** [comment]
Hi @katungi 👋 

The relation field `steps`  define the connections between models at the Prisma level and do not exist in the database. These fields are used to generate the Prisma Client. You can learn more about this [here](https://www.prisma.io/docs/concepts/components/prisma-schema/relations#relation-fields:~:text=Two%20relation%20fields%3A%20author%20and%20posts.%20Relation%20fields%20define%20connections%20between%20models%20at%20the%20Prisma%20level%20and%20do%20not%20exist%20in%20the%20database.%20These%20fields%20are%20used%20to%20generate%20the%20Prisma%20Client.)

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise, please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏
